tests/syscall/fs_linux/enc/file_system.h

```diff
@@ -0,0 +1,578 @@
+// Copyright (c) Open Enclave SDK contributors.
+// Licensed under the MIT License.
+
```

Comment:
I can't tell what, if anything, in this file is actually Linux specific, nor if anything is SGX-specific, even though I see "linux" in the directory name this file is under, and SGX in the content.  I also agree with Simon's comments.

Reply:
Fixed with Simon's suggestion, thanks.